releases/release-1.31/release-notes/maps/pr-124328-map.yaml

```
@@ -0,0 +1,3 @@
+pr: 124328
+releasenote:
+  text: The name of CEL optional type has been changed from `optional` to `optional_type`.
```

```suggestion
  text: The name of CEL(Common Expression Language) optional type has been changed from `optional` to `optional_type`.
```

https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653206958